charts/kyverno-policies/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: kyverno-policies
-version: v2.5.5
+version: v2.6.0
```

Comment:
Let's not bump the version in this PR, I'll change it once cherry-pick it to release-1.8.